Add `/ess dump` command to generate a debug dump output

JRoy opened the pull request:
Command Usage;
/essentials dump [config] [discord] [kits] [log]

Either of the optional args can be used to add the given data to the dump.

Dependent on https://github.com/EssentialsX/Website/pull/51

JRoy committed:
Move paste operations to util class

JRoy committed:
Add /essentials dump command

JRoy committed:
Use essentials website dump parser

JRoy committed:
Add usage string

JRoy committed:
Sort plugins alphabetically

JRoy committed:
Add wildcard option for dump

JRoy committed:
Merge branch '2.x' into feature/dump

JRoy committed:
Merge branch '2.x' into feature/dump

JRoy committed:
Merge branch '2.x' into feature/dump

JRoy committed:
Merge branch '2.x' into feature/dump

LaxWasHere reviewed (dismissed):
![waitin](https://media3.giphy.com/media/tXL4FHPSnVJ0A/source.gif)

JRoy committed:
Merge branch '2.x' into feature/dump

JRoy committed:
Merge branch '2.x' into feature/dump

mdcfe committed:
Add metadata and more plugin info

- Adds `meta` with command sender info and dump creation timestamp
- Adds plugin author and descriptions to payload
- Marks the core Essentials plugin as "official"

mdcfe committed:
Log deletion key along with the dump URL

mdcfe commented on Essentials/src/main/resources/messages.properties:
```diff
@@ -309,6 +315,8 @@ essentialsCommandUsage6=/<command> cleanup
 essentialsCommandUsage6Description=Cleans up old userdata
 essentialsCommandUsage7=/<command> homes
 essentialsCommandUsage7Description=Manages user homes
+essentialsCommandUsage8=/<command> dump [config] [discord] [kits] [log]
```
```suggestion
essentialsCommandUsage8=/<command> dump [*] [config] [discord] [kits] [log]
```

mdcfe committed:
Add asterisk to usage message

mdcfe committed:
Merge branch '2.x' into feature/dump
